back/cmd/openbsrv/srvmgmt.go

```diff
@@ -32,7 +34,7 @@ func (m *serverMgmt) serve() error {
 
 			// TODO: gather returned errors
 			if err := s.Serve(); err != nil {
-				fmt.Fprintln(os.Stderr, "server error:", err)
+				m.log.Error("server error: %v", err)
```

Comment:
This function (`serve()`) and `stop()` should be returning the accumulated errors as noted in the TODOs. Please create an issue for that if one does not exist.

Reply:
Please undo this change back to fmt.print to os.stderr.

Reply:
Issue #160 has been filed.